pallets/phala/src/stakepool.rs

```diff
@@ -540,6 +562,110 @@ pub mod pallet {
 
 			Ok(())
 		}
+		/// add a staker accountid to contribution whitelist.
```

Comment:
Capitalize and spaces. Please refer to [docstring style guide](https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html#documenting-components).

Reply:
done